src/app/component/Swipeable/Swipeable.style.scss

```diff
@@ -0,0 +1,31 @@
+.Swipeable {
+    overflow: hidden;
+
+    &-Container {
+        display: flex;
+    }
+
+    &-Panel {
+        min-height: 100%;
+        min-width: 100%;
+        background-color: #eb4444;
```

Comment:
Please, add declaration on `:root` for CSS variable color. Use as variable here.